reaper/reaper/src/main/kotlin/com/emergetools/reaper/ReaperInternal.kt

```diff
@@ -128,6 +129,11 @@ internal object ReaperInternal {
       fatalError(context, "Manifest com.emergetools.PUBLISHABLE_API_KEY must be set and non-empty.")
     }
 
+    isDebug = context.packageManager.getApplicationInfo(
+      context.packageName,
+      PackageManager.GET_META_DATA
+    ).metaData.getBoolean(MANIFEST_TAG_DEBUG, false)
```

Comment:
What's the plan here, assuming there will be a grade plugin tie-in to set this tag from an extension property?

Reply:
So far I've been manually adding:
```
<meta-data
   android:name="com.emergetools.reaper.DEBUG"
   android:value="true" />
```
to the manifest when needed. Would be neat to hook it up to the grade plugin though. I think the plugin already exposes a 'debug' setting iirc